dubbo-common/src/main/java/org/apache/dubbo/common/compiler/support/ClassUtils.java

```diff
@@ -1,445 +1,445 @@
-                        || type == float.class
-                        || type == double.class) {
-                    def = "0";
-                } else {
-                    def = "null";
-                }
-                buf.append(def);
-            }
-        }
-        return method.getName() + "(" + rightCode + ")";
-    }
-
-    public static Method searchMethod(Class<?> currentClass, String name, Class<?>[] parameterTypes) throws NoSuchMethodException {
-        if (currentClass == null) {
-            throw new NoSuchMethodException("class == null");
-        }
-        try {
-            return currentClass.getMethod(name, parameterTypes);
-        } catch (NoSuchMethodException e) {
-            for (Method method : currentClass.getMethods()) {
-                if (method.getName().equals(name)
-                        && parameterTypes.length == method.getParameterTypes().length
-                        && Modifier.isPublic(method.getModifiers())) {
-                    if (parameterTypes.length > 0) {
-                        Class<?>[] types = method.getParameterTypes();
-                        boolean match = true;
-                        for (int i = 0; i < parameterTypes.length; i++) {
-                            if (!types[i].isAssignableFrom(parameterTypes[i])) {
-                                match = false;
-                                break;
-                            }
-                        }
-                        if (!match) {
-                            continue;
-                        }
-                    }
-                    return method;
-                }
-            }
-            throw e;
-        }
-    }
-
-    public static String getInitCode(Class<?> type) {
-        if (byte.class.equals(type)
-                || short.class.equals(type)
-                || int.class.equals(type)
-                || long.class.equals(type)
-                || float.class.equals(type)
-                || double.class.equals(type)) {
-            return "0";
-        } else if (char.class.equals(type)) {
-            return "'\\0'";
-        } else if (boolean.class.equals(type)) {
-            return "false";
-        } else {
-            return "null";
-        }
-    }
-
-    public static <K, V> Map<K, V> toMap(Map.Entry<K, V>[] entries) {
-        Map<K, V> map = new HashMap<K, V>();
-        if (entries != null && entries.length > 0) {
-            for (Map.Entry<K, V> entry : entries) {
-                map.put(entry.getKey(), entry.getValue());
-            }
-        }
-        return map;
-    }
-    
-    /**
-     * get simple class name from qualified class name
-     */
-    public static String getSimpleClassName(String qualifiedName) {
-        if (null == qualifiedName) {
-            return null;
-        }
-        
-        int i = qualifiedName.lastIndexOf('.');
-        return i < 0 ? qualifiedName : qualifiedName.substring(i + 1);
-    }
-
-}
+/*
```

Comment:
Pls recover end-line format of this file

Reply:
it's CRLF to LF problem. recover it already.